jenkins/jobs/jobs.yaml

```diff
@@ -175,6 +175,49 @@
       - osci-swift-upload-logs-with-console-plugin:
           upload_source: 'test.log logs'
 
+- job-template:
+    name: 'dsvm-tempest-vgpu'
+    node: '{node}'
+
+    wrappers:
+      - timeout:
+          timeout: 30  # Timeout in *minutes*
+          fail: true  # A job run that exceeds the timeout will cause a failure
+      - timestamps
+
+    builders:
+      #- net-info
+      - shell: |
+          #!/bin/bash -x
+          env
+
+          #If disable ceilometer by force
+          export FORCE_DISABLE_CEILOMETER=""
+          export ZUUL_CHANGES=${ZUUL_CHANGES:-""}
+          export ZUUL_REF=${ZUUL_REF:-""}
+          export ZUUL_CHANGE=${ZUUL_CHANGE:-""}
+          export OPENSTACK_LOGS=/tmp/opentack_logs
```

Comment:
Can we put openstack logs under $WORKSPACE/logs? then no need the following line to upload logs:
cp $OPENSTACK_LOGS logs/ -rf

Reply:
Will do it